### PR TITLE
fix: small fixes ui 3

### PR DESCRIPTION
**Pull request by tonysnowboardunderthebridge**
fix: updating some points for the fixes on Compose Application and copy Logs

### CODE REVIEW

File: src/renderer/src/components/steps/GeneratingApplication.jsx [hunk 1]
```diff
@@ -61,9 +62,20 @@ const GeneratingApplication = React.forwardRef(({ onBack, onRestartProcess, useV
   }
 
   function onClickCopyLogs () {
+    setLogsCopied(true)
     let str = ''
     npmLogs.forEach(log => (str += `${log.message}\r\n`))
     navigator.clipboard.writeText(str)
+    setTimeout(() => {
```
Comment (marcopiraccini) [inline src/renderer/src/components/steps/GeneratingApplication.jsx:69]: Why this? `writeText` is async. https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/writeText

File: src/renderer/src/components/steps/UpdatingApplication.jsx [hunk 2]
```diff
@@ -60,9 +62,20 @@ const UpdatingApplication = React.forwardRef(({ onBack, onClickGoToApps, applica
   }
 
   function onClickCopyLogs () {
+    setLogsCopied(true)
     let str = ''
     npmLogs.forEach(log => (str += `${log.message}\r\n`))
     navigator.clipboard.writeText(str)
+    setTimeout(() => {
```
Comment (marcopiraccini) [inline src/renderer/src/components/steps/UpdatingApplication.jsx:69]: Same here

**Review by marcopiraccini — CHANGES_REQUESTED** [review]

File: src/renderer/src/components/steps/GeneratingApplication.jsx (hunk 1 above)
Comment (tonysnowboardunderthebridge) [reply src/renderer/src/components/steps/GeneratingApplication.jsx:69]: I added this setTimeout to change the icon

**Review by marcopiraccini — APPROVED** [review]
OK

**Pull request merged**